other/gauss_easter.py

```diff
@@ -56,5 +56,5 @@ def gauss_easter(year: int) -> datetime:
 
 if __name__ == "__main__":
     for year in (1994, 2000, 2010, 2021, 2023):
```

Comment:
Perhaps add some future years.
```suggestion
    for year in (1994, 2000, 2010, 2021, 2023, 2032, 2100):
```